detail.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+import math
+
```

Comment:
Do you need math/date time imported here?

Reply:
No. Thanks for your valuable input MatthewDembinski21